tests/nosetests/pyanaconda_tests/module_source_nfs_test.py

```diff
@@ -114,6 +114,13 @@ def url_property_test(self):
         self.module.set_url(nfs_url)
         self.assertEqual(nfs_url, self.module.url)
 
+    def repr_test(self):
+        self.module.set_url(nfs_url)
+        self.assertEqual(
+            repr(self.module),
+            "Source(type='NFS', url='{}')".format(nfs_url)
```

Comment:
Please, don't use the final string instead of the formatting.

Reply:
You mean, I should get "NFS" there via `format` too?

Reply:
My mistake. It should be "use the final string instead of the formatting". It is better to be sure that the output really matches the expected value.

Reply:
ACK, will do.